using-timescaledb/compression.md

```diff
@@ -19,184 +19,237 @@ basics of setting this up for use in your environment.
+
+The `orderby` effects both the compression ratio achieved and the query performance.
+
+Compression is most effective when adjacent data is close in magnitude or exhibits
+some sort of trend. In other words, random or out of order data will compress poorly.
+Thus, when compressing data it is important that the order of the input data 
+causes it to follow a trend.
+
+Let's look again at what our running example looked like without any segmentby columns.
+
+|time|device_id|cpu|disk_io|energy_consumption|
+|---|---|---|---|---|
+| [12:00:02, 12:00:02, 12:00:01, 12:00:01 ]| [1, 2, 1, 2]|[88.2, 300.5, 88.6, 299.1]|[20, 30, 25, 40] |[0.8, 0.9, 0.85, 0.95]| 
+
+Notice that the data is ordered by the `time` column. But, if we look at the
+`cpu` column, we can see that the compressor will not be able to efficiently
+compress it. Although both devices output a value that is a float, the
+measurements have different magnitudes. The float list [88.2, 300.5, 88.6,
+299.1] will compress poorly because values of the same magnitude are not
+adjacent. However,  we can order by `device_id, time DESC` by setting
+our table options as follows:
+
+``` sql
+ALTER TABLE  measurements 
+  SET (timescaledb.compress, 
+       timescaledb.compress_orderby = 'device_id, time DESC');
+```
+
+Using those settings, the compressed table will look as follows:
+
+|time|device_id|cpu|disk_io|energy_consumption|
+|---|---|---|---|---|
+| [12:00:02, 12:00:01, 12:00:02, 12:00:01 ]| [1, 1, 2, 2]|[88.2, 88.6, 300.5, 299.1]|[20, 25, 30, 40] |[0.8, 0.85, 0.9, 0.95]|
+
+Now, each devices measurement is consecutive in the ordering and
+and thus the measurement values exhibit more of a trend. The cpu
+series [88.2, 88.6, 300.5, 299.1] will compress much better.
+
+If you look at the above example with `device_id` as a `segmentby`,
+you will see that this will have good compression as well since
+ordering only matters within a segment and segmenting by device
+guarantees that each segment represents a series if only ordered by time.
+Thus, putting items in `orderby` and `segmentby` columns achieves similar
+results. This is why, if segmenting by a identifier causes segments to become too
+small, we recommend moving the segmentby column into a prefix of the
+orderby list.
+
+We also use ordering to increase query performance. If a query uses the same
+(or similar) ordering as the compression, we know that we can decompress
+incrementally and still return results in the same order. We can also 
+avoid a sort. In addition, the system automatically creates additional columns
+that store the minimum and maximum value of any `orderby` column. This way, the
+query executor can look at this special column that specifies the range of
+values (e.g. timestamps) in the compressed column – without first performing any
+decompression – in order to determine whether the row could possibly match a
+time predicate specified by a user’s SQL query.
 
 ---
```

Comment:
With the headers in the appropriate hierarchy, these dividers can be removed.  Also, they aren't applied consistently